tce/src/applibs/wxToolkit/MachineWindow/MachineCanvas.cc

```diff
@@ -523,6 +523,7 @@ bool
 MachineCanvas::saveEPS(const std::string& filename, const std::string& title,
     const std::string& creator) {
 
+#if !wxCHECK_VERSION(3, 0, 0)
```

Comment:
Better comment out the whole method and also remove the possible menu items related to this until it works.